plugins/modules/network/routeros/routeros_api.py

```diff
@@ -0,0 +1,489 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Nikolay Dachev <nikolay@dachev.info>
+# GNU General Public License v3.0+ https://www.gnu.org/licenses/gpl-3.0.txt
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: routeros_api
+version_added: 1.1.0
+author: Nikolay Dachev (@nikolaydachev)
+short_description: Ansible module for RouterOS API
+description:
+  - Ansible module for RouterOS API with python librouteros.
+     This module can add, remove, update, query and execute arbitrary command
+     in routeros via api
+options:
+  hostname:
+    description:
+      - RouterOS hostname api
```

Comment:
```suggestion
      - RouterOS hostname api.
```
it's fair for all descriptions.